internal/core/data/v2/handler/io/event.go

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
Can we more `io` package up one level so it isn't in `handler`?

Reply:
Updated the `io` package to one level up as suggested.